config/config.exs

```diff
@@ -6,7 +6,7 @@ config :ex_plasma,
   authority_address: "0x22d491bde2303f2f43325b2108d26f1eaba1e32b",
   contract_address: "0xd17e1233a03affb9092d5109179b43d6a8828607",
   eth_vault_address: "0x1967d06b1faba91eaadb1be33b277447ea24fa0e",
-  exit_game_address: "0xf2507a7ed2c706e64c3e84acb552ee71bc9522d4",
+  exit_game_address: "0x2a7be4e7f65afbc6e8f9b1d522fb403e9202363b",
```

Comment:
similar to `elixir-omg`, exit game address should be derived from the payment type